src/models/environment-variable/editable-environment-variable.js

```diff
@@ -0,0 +1,7 @@
+import PropTypes from 'prop-types';
+import EnvVarModel from '.';
+
+export default Object.freeze({
```

Comment:
Seems like this isnot used anywhere. Can it be removed?

Reply:
Not anymore. Deleted.